src/main/java/uk/gov/di/ipv/cri/common/library/domain/personidentity/SharedClaims.java

```diff
@@ -1,10 +1,15 @@
 package uk.gov.di.ipv.cri.common.library.domain.personidentity;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 import java.util.List;
 
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
I think this should also probably have @JsonInclude(JsonInclude.Include.NON_NULL)